.github/workflows/_run_e2e_test.yml

```diff
@@ -0,0 +1,44 @@
+name: Common Steps
```

Comment:
nits: 

`Run E2E Test Common Steps` or something?

Reply:
Thanks for the suggestion. I've fixed it.👍
https://github.com/liam-hq/liam/pull/660/commits/b8f108b5e741e1974c03039fbcebee2392f20d98